scripts/techsupport

```diff
@@ -47,10 +47,14 @@ if [[ $1 == 'reset' ]]; then
 else
   top -n 10 -b > top-output.txt
   rsync -a /home/pi/.firewalla ./ --exclude run/
+  rsync -a /home/pi/.router ./ --exclude run/ --exclude tmp/ --exclude config/sshd/keys
   # remove unnecessary files
   rm -f ./.sshpasswd
   rm -f ./.firewalla/run/ovpn_profile/*.password
   rm -r ./.firewalla/run/ovpn_profile/*.userpass
+  sed -i -E 's/^(\s*(user|password)) .*$/\1 ****/' .router/config/pppoe/*
+  sed -i -E 's/^(\s*(ssid|.*identity|.*password|psk|wep_key.|phase.|.*_passwd))=.*$/\1=****/' .router/config/wpa_supplicant/*
```

Comment:
Just to confirm,

wep_key. NOT wep_key.*  ？
phase. NOT phase.* ?

Reply:
nope